serving/features.go

```diff
@@ -0,0 +1,284 @@
+package serving
+
+import (
+	"context"
+	"fmt"
+	"github.com/featureform/metadata"
+	"github.com/featureform/metrics"
+	pb "github.com/featureform/proto"
+	"github.com/featureform/provider"
+	pt "github.com/featureform/provider/provider_type"
+	"sort"
+)
+
+type indexedValue struct {
+	index int
+	value interface{}
+}
+
+type indexedFeatureRow struct {
+	index  int
+	values *pb.ValueList
+}
+
+func (serv *FeatureServer) getFeatureRows(ctx context.Context, features []*pb.FeatureID, entityMap map[string][]string) ([]*pb.ValueList, error) {
```

Comment:
does go format function parms so it's not just one huge line?

Reply:
Don't think so lol